src/test/java/acceptance/queue/QueueSteps.java

```diff
@@ -1,57 +1,80 @@
 package acceptance.queue;
 
-import cucumber.api.java.en.*;
 import acceptance.SingletonTestBroker;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.But;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import tdl.client.audit.StdoutAuditStream;
 import tdl.client.queue.ImplementationRunnerConfig;
 import tdl.client.queue.QueueBasedImplementationRunner;
 import tdl.client.queue.abstractions.UserImplementation;
 import tdl.client.queue.actions.ClientAction;
 import tdl.client.queue.actions.ClientActions;
-import tdl.client.audit.StdoutAuditStream;
-import utils.jmx.broker.RemoteJmxQueue;
+import tdl.client.runner.connector.EventProcessingException;
+import tdl.client.runner.connector.QueueEventHandlers;
+import tdl.client.runner.connector.SqsEventQueue;
+import tdl.client.runner.events.ExecuteCommandEvent;
 import utils.logging.LogAuditStream;
 
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
-import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.Matchers.not;
 import static org.junit.Assert.assertThat;
 
 public class QueueSteps {
     private final SingletonTestBroker broker;
 
     // Test broker location
     private static final String HOSTNAME = "localhost";
-    private static final int PORT = 21616;
+    private static final int PORT = 28161; // 21616
 
     // Variables set by the background tasks
-    private RemoteJmxQueue requestQueue;
-    private RemoteJmxQueue responseQueue;
+    private SqsEventQueue requestQueue;
```

Comment:
We should not use this at all.
Also the test code should not share code with production code.

Reply:
This might have been remnant from previous attempts to get the queue to work